manifest.json

```diff
@@ -4,10 +4,10 @@
     "packaging_format": 1,
     "license": "LGPL-2.1,GPL-2.0",
     "url": "https://sogo.nu/",
-    "version": "4.0.7-1~ynh2",
+    "version": "5.0.1~ynh1",
```

Comment:
Well we use the debian package from debian so it still be the version 4.0.7 on debian buster.

Reply:
ah, this is because I am using Bullseye...

Reply:
Yes but as I know actually we still be on buster on the stable version...